src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -4,22 +4,25 @@
 </style>
 <html>
 <head>
-    <title>My team</title>
+    <title>Redirect page</title>
 </head>
 <body>
+<%@include file="header.jsp"%>
 <form method="post" id="redirect"></form>
-<h1 class="table_dark">Hello, mates</h1>
+<h1 class="table_dark">Hello, driver</h1>
 <table class="table_dark">
     <tr>
         <th>Redirect to</th>
     </tr>
+    <tr><td><a href="${pageContext.request.contextPath}/drivers/login">Login</a></td></tr>
     <tr><td><a href="${pageContext.request.contextPath}/drivers/">Display All Drivers</a></td></tr>
     <tr><td><a href="${pageContext.request.contextPath}/cars/">Display All Cars</a></td></tr>
     <tr><td><a href="${pageContext.request.contextPath}/manufacturers/">Display All Manufacturers</a></td></tr>
-    <tr><td><a href="${pageContext.request.contextPath}/drivers/add">Create new Driver</a></td></tr>
-    <tr><td><a href="${pageContext.request.contextPath}/cars/add">Create new Car</a></td></tr>
-    <tr><td><a href="${pageContext.request.contextPath}/manufacturers/add">Create new Manufacturer</a></td></tr>
+    <tr><td><a href="${pageContext.request.contextPath}/drivers/add">Register new Driver</a></td></tr>
+    <tr><td><a href="${pageContext.request.contextPath}/cars/add">Register new Car</a></td></tr>
```

Comment:
Is it works fine? You dont have url drivers/add in web it has another name

Reply:
Yes, I have url in web.xml. Maybe you didn't see)

Reply:
No, I didn't see, I fixed it)